src/bin/travis/afterSuccess.sh

```diff
@@ -9,7 +9,12 @@ echo "TRAVIS_PULL_REQUEST='$TRAVIS_PULL_REQUEST'"
 if [ "$BUILD_TYPE" == "sbt" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
 
     if [ "$TRAVIS_BRANCH" == "develop" ]; then
-        sbt 'set test in Test := {}' publish
+        docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+        sbt \
+          'set test in Test := {}' \
```

Comment:
Why does Test need to be explicitly stated as an empty set? Ive seen it in multiple places now but I'm unclear of the why